anndata/tests/test_dask.py

```diff
@@ -93,3 +93,19 @@ def test_assign_X(adata):
 
 
 # TODO: Views
+
+def test_assert_eq(adata):
+
+    import dask.array as da
+    import numpy as np
+    from dask.array.utils import assert_eq
```

Comment:
I think this is just for demonstration, right? Because we don't use the adata argument. Anyway, it seems like this function should be enough, so I will add it to the dispatch flow.

Reply:
yup, only for demo